proposals/0000-exception-backtraces.rst

```diff
@@ -74,7 +74,7 @@ To avoid breaking existing users of ``SomeException``, we introduce this
 context as an implicit parameter constraint:  ::
 
     data SomeException where
-        SomeException :: forall a. (Exception a, ?exceptionContext :: ExceptionContext)
+        SomeException :: forall a. (Exception a, ?context :: ExceptionContext)
```

Comment:
This could conflict with an application-specific implicit parameter named `?context`. Why not keep it as `?exceptionContext`?

Reply:
Sigh, I could have sworn that there was a suggestion to change this to `?context` earlier in the discussion and which I somewhat mindlessly followed. However, I share your concern; I'll change this to `exceptionContext`.